dipy/core/gradients.py

```diff
@@ -127,6 +194,25 @@ def gradient_table_from_bvals_bvecs(bvals, bvecs, b0_threshold=50, atol=1e-2,
     atol : float
         Each vector in `bvecs` must be a unit vectors up to a tolerance of
         `atol`.
+    btens : can be any of three options
```

Comment:
I think the supported types still need to be summarized next to the variable name.

Reply:
I followed the conventions that had been used in docstrings of other functions like gradient_table_from_qvals_bvecs.

Reply:
I see. Thanks. I'd then be for changing them in another PR, but I admit the difficulty of proposing a compact notation for all cases.